File: 1008/main.cc
#include <stdio.h>
#include <iostream>
#include <iomanip>

using namespace std;
 
int main() {
 
    int numFunc, horasTrab;
    double valorHora, salario;
    cin >> numFunc >> horasTrab >> valorHora;
    salario = valorHora * horasTrab;
    cout << "NUMBER = " << numFunc << endl << "SALARY = U$ " << fixed << setprecision(2) << salario << endl;
 
    return 0;
}

File: 1008/main.java
import java.io.IOException;
import java.util.Scanner;
 

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        int num, horasTrab;
        double valorHora, salario;
        
        Scanner input = new Scanner(System.in);
        
        num = input.nextInt();
        horasTrab = input.nextInt();
        valorHora = input.nextDouble();
        
        salario = horasTrab * valorHora;
        
        System.out.println("NUMBER = " + num);
        System.out.println(String.format("SALARY = U$ %.2f" , salario));
    }
 
}